_includes/schemasorg.html

```diff
@@ -0,0 +1,30 @@
+<script type="application/ld+json">
+    {
+        "@context": "https://schema.org",
+        "@type": "WebSite",
+        "name": "{{ site.title }}",
+        "url": "{{ site.url }}",
+        "about": "{{ site.description }}",
+        {%- if page.contributors %}
+        {%- assign contributors = site.data.CONTRIBUTORS -%}
+        {% for people in page.contributors %}
```

Comment:
I will have expected something like
```
"contributor": [
    {
      "@type": "Person",
      "@id": "https://orcid.org/0000-0003-0522-5674",
      "mainEntityOfPage": "https://orcid.org/0000-0003-0522-5674",
      "givenName": "Bert",
      "familyName": "Droesbeke",
      "affiliation": {
        "@type": "Organization",
        "name": "VIB-UGent / ELIXIR-BE"
      },
      "email": "bedro@psb.ugent.be",
      "url": "https://github.com/bedroesb/",
      "image": "https://avatars.githubusercontent.com/bedroesb",
      "jobTitle": "editor"
    },
```
``orcid`` is for example not available in schema.org

Reply:
@jburel Thanks for your comment! 

I used orcid from Bioschemas: https://bioschemas.org/profiles/Person/0.2-DRAFT-2019_07_19

But using it as ID makes also sense. Do you think it is a problem to leave out:

```
"@id": "https://orcid.org/0000-0003-0522-5674",
      "mainEntityOfPage": "https://orcid.org/0000-0003-0522-5674",
```
When no orcid is given?

Second question, The name of the person is at this moment name + family name in one string. I don't think I can split this in an automatic way.

Reply:
The only downside is that it is still a draft so we might be careful about. 
In the case of orcid is not there, we will omit ID (not ideal). Is there a field we are sure to always have?

For the name, if I understand correctly, ``givenName`` is currently misused. Should the ``name`` field of ``Person`` used in that case to avoid misinterpretation.

Reply:
Name (first + last name) is what we have for sure. I will make a commit implementing these suggestions, thanks!